tests/utils/test_maven.py

```diff
@@ -0,0 +1,119 @@
+"""Test for maven.py
+
+Test for the utility for maven operations.
+"""
+from testfixtures import TempDirectory
+from tests.helpers.base_tssc_test_case import BaseTSSCTestCase
+from tssc.utils.maven import generate_maven_settings
+
+
+class TestMavenUtils(BaseTSSCTestCase):
+    # def test_generate_maven_settings_maven_servers_is_none(self):
+
+    def test_generate_maven_servers_exist(self):
+        maven_servers = [
+            {
+                "id": "one",
+                "username": "user1",
+                "password": "password1"
+            }
+        ]
+
+        settings = '''<settings><servers><server><id>one</id><username>user1</username><password>password1</password></server></servers></settings>'''
+
+        with TempDirectory() as temp_dir:
+            generate_maven_settings(temp_dir.path, maven_servers, None, None)
+            with open(temp_dir.path + '/settings.xml', 'r') as tester:
+                results = tester.read()
+                assert results == settings
+
+    def test_generate_maven_servers_id_does_not_exist(self):
+        maven_servers = [
+            {
+                "username": "user1",
+                "password": "password1"
+            }
+        ]
+        with TempDirectory() as temp_dir:
+            with self.assertRaisesRegex(ValueError, 'id is required for maven_servers.'):
+                generate_maven_settings(temp_dir.path, maven_servers, None, None)
+
+    def test_generate_maven_servers_username_does_not_exist(self):
+        maven_servers = [
+            {
+                "id": "one",
+                "password": "password1"
+            }
+        ]
+        with TempDirectory() as temp_dir:
+            with self.assertRaisesRegex(ValueError,
+                                        'username and password are required for maven_servers.'):
+                generate_maven_settings(temp_dir.path, maven_servers, None, None)
+
+    def test_generate_maven_repositories_exists(self):
```

Comment:
Do you have a test for when repositories exist me are empty?

Reply:
@akim309 - passing this one to you

Reply:
@akim309  Your tests currently cover the step properly. You can ignore this request.